test/e2e/tests/ppom/ppom-blockaid-alert-erc20-transfer.spec.js

```diff
@@ -181,10 +180,10 @@ describe('PPOM Blockaid Alert - Malicious ERC20 Transfer @no-mmi', function () {
       async ({ driver }) => {
         const expectedTitle = 'This is a deceptive request';
         const expectedDescription =
-          'If you approve this request, you might lose your assets.';
```

Comment:
the malicious descriptions have slightly changed so we need to modify them in the specs